source/Octopus.Tentacle/Commands/RunAgentCommand.cs

```diff
@@ -101,6 +102,7 @@ protected override void Start()
             Environment.SetEnvironmentVariable(EnvironmentVariables.TentacleProgramDirectoryPath, Path.GetDirectoryName(exePath));
             Environment.SetEnvironmentVariable(EnvironmentVariables.AgentProgramDirectoryPath, Path.GetDirectoryName(exePath));
             Environment.SetEnvironmentVariable(EnvironmentVariables.TentacleVersion, appVersion.ToString());
+            Environment.SetEnvironmentVariable(EnvironmentVariables.TentacleNetFrameworkDescription, RuntimeInformation.FrameworkDescription);
```

Comment:
This will allow Server to ask Tentacle, via a script, what framework it is running under, which may be needed for the changes in Server.

By adding it now, we guarantee that all Windows net6.0 installations have this environment variable set so we can rely on its existence.